### PR TITLE
Prac 06 feedback

### PR DESCRIPTION
**Pull request by Liam-Eime**
Please review my code @km-aung560 checking for best practices and such :)

### CODE REVIEW

File: prac_06/car.py [hunk 1]
```diff
@@ -4,14 +4,18 @@
...
 
-    def __init__(self, fuel=0):
+    def __init__(self, name="", fuel=0):
         """Initialise a Car instance.
 
         fuel: float, one unit of fuel drives one kilometre
         """
+        self.name = name
         self.fuel = fuel
         self._odometer = 0
```
Comment (km-aung560) [inline prac_06/car.py:15]: Seems like you are missing a 'drive' function...

File: prac_06/programming_language.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+CP1404 | Practical 06 - programming_language and languages  | Liam Eime
+Estimate: 20 minutes
+Actual:
+"""
+
+
+class ProgrammingLanguage:
+    """Represent programming language information"""
+
+    def __init__(self, name="", typing="", reflection=False, year=0):
```
Comment (km-aung560) [inline prac_06/programming_language.py:11]: For your reflection object, are you sure you want the default value to only false? Maybe there is something better to set the value to.
Also, the same applies to the default value of the year object.

File: prac_06/guitar.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        """Return guitar information"""
+        return f"{self.name} ({self.year}) : ${self.cost:,.2f}"
+
+    def get_age(self):
+        """Return how old the guitar is"""
+        return CURRENT_YEAR - self.year
+
+    def is_vintage(self):
+        """Return True if the guitar is 50 or more years old"""
+        if self.get_age() >= 50:
```
Comment (km-aung560) [inline prac_06/guitar.py]: It would be better if you could return the expression itself rather than having if-else.

File: prac_06/car.py (hunk 1 above)
Comment (Liam-Eime) [reply prac_06/car.py:15]: No, it is there? line 19

File: prac_06/guitar.py (hunk 3 above)
Comment (Liam-Eime) [reply prac_06/guitar.py]: Yes, I made that change already in a commit later on. Check the commit after this one!

**Review by Liam-Eime — COMMENTED** [review]
Followed up on some comments :)

File: prac_06/guitar.py (hunk 3 above)
Comment (km-aung560) [reply prac_06/guitar.py]: Sorry about that! I was a bit confused with the commit files 😆

**Pull request merged**